docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,46 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Improve visibility on how the decisions made, producing a more open community mood
+ * Clarity on what the proposal is, its rationale and impact
+ * Reduce unnecessary work when coding large features is done without community and core-committers buy-in
+ * Improved likelihood of an optimal solution being merged - the pressure is much higher to approve the change if the coding has already been done (it’s much easier to make adjustments in the design phase)
```

Comment:
Its unclear if "pressure much higher" is a positive or negative sentiment here. I'd just keep the first sentence, the second one is already contained in the previous bullet point